doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,117 @@
+.. _ref_make_container:
+
+Create your Fluent Docker container
+===================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps detailed in this section.
+
+You can create your Fluent Docker container by following the steps given on this page.
+This guide will use a local Ubuntu machine to copy the needed files from the Fluent 
```

Comment:
```suggestion
The following procure for creating a Fluent Docker container uses a local Ubuntu machine to copy the needed files from the Fluent 
```
The original text was a bit redundant with the warning.

Reply:
Done. Thank you.